### ISSUE TITLE
Documentation misses pkg-config dependency for OSX

### ISSUE THREAD

**Issue by devrando** [borgbackup/borg#1400]
Following:
http://borgbackup.readthedocs.io/en/stable/installation.html#git-installation
and installing deps:
```
brew install python3 lz4 openssl
pip3 install virtualenv
```
Building from Source fails at:

`
pip install -r requirements.d/fuse.txt  # optional, for FUSE support
`
with

```
    FileNotFoundError: [Errno 2] No such file or directory: 'pkg-config'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/1r/k62qvc1j3nn8w7mrywj0g4h40000gn/T/pip-build-p4ol2zmk/llfuse/
```

So documentation should be:

```
brew install python3 lz4 openssl pkg-config
pip3 install virtualenv

```

EDIT: Duplicate! Sorry @enkore you've been 5 minutes faster ;)

**Comment by PlasmaPower** [comment]
> brew install python3 lz4 openssl pkg-config

Our Vagrantfile has `brew install pkgconfig`, without a dash. Can you check which is correct, or if both are?

**Comment by PlasmaPower** [comment]
Odd, it looks like `pkg-config` is correct. Most likely our Vagrantfile pulls it in anyways as part of something else and silently fails.

**Comment by ThomasWaldmann** [comment]
Obviously both names work, but pkg-config seems to be the canonical name:
```
This-MacBook-Pro:~ vagrant$ brew info pkg-config
pkg-config: stable 0.29.1 (bottled)
Manage compile and link flags for libraries
https://freedesktop.org/wiki/Software/pkg-config/
/usr/local/Cellar/pkg-config/0.29.1_1 (10 files, 627.3K) *
  Poured from bottle on 2016-07-30 at 11:31:30
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/pkg-config.rb
```
```
This-MacBook-Pro:~ vagrant$ brew info pkgconfig
pkg-config: stable 0.29.1 (bottled)
Manage compile and link flags for libraries
https://freedesktop.org/wiki/Software/pkg-config/
/usr/local/Cellar/pkg-config/0.29.1_1 (10 files, 627.3K) *
  Poured from bottle on 2016-07-30 at 11:31:30
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/pkg-config.rb
```

**Comment by ThomasWaldmann** [comment]
Fixed by PR #1409 in 1.0-maint, soon in master.
